src/Plugin/AbstractFX.hpp

```diff
@@ -178,6 +222,20 @@ class AbstractPluginFX : public Plugin
         else
             multiply(outputs[1], 0.5f, frames);
 
+        if(timePosition.bbt.valid) {
```

Comment:
General question: is timePosition.bbt.valid true for all frames in a host that has valid time position information or is there the risk that it's only valid for some frames?